iapyx/src/wallet.rs

```diff
@@ -30,15 +30,15 @@ pub struct Wallet {
 impl Wallet {
     pub fn recover(secret_key: &[u8]) -> Result<Self, Error> {
         Ok(Self {
-            inner: Inner::recover_free_keys(secret_key, [].iter())
+            inner: Inner::recover_free_keys(secret_key)
                 .map_err(|e| Error::CannotRecover(e.to_string()))?,
             pending_txs: Vec::new(),
         })
     }
 
     pub fn recover_from_utxo(secret_key: &[u8; 64]) -> Result<Self, Error> {
         Ok(Self {
-            inner: Inner::recover_free_keys(secret_key, [*secret_key].iter())
```

Comment:
is there a test for this?

Reply:
I'm not sure, though this repo is mainly testing tools, so arguably this repo *is* the test 🤷 This was only needed since we needed to update downstream code, which changed some API signatures

Reply:
ok cool makes sense